kolibri/core/assets/src/api-resources/task.js

```diff
@@ -1,9 +1,68 @@
 const Resource = require('../api-resource').Resource;
+const logging = require('logging').getLogger(__filename);
 
 class TaskResource extends Resource {
   static resourceName() {
     return 'task';
   }
+
+  localExportContent(driveId) {
+    const promise = new Promise((resolve, reject) => {
+      const clientObj = { path: this.localExportUrl(), entity: { id: driveId },
+        headers: { 'Content-Type': 'application/json' } };
```

Comment:
Change the `.wrap(mime)` to: `.wrap(mime, { mime: 'application/json' })` in this line:
https://github.com/learningequality/kolibri/blob/master/kolibri/core/assets/src/api-resource.js#L500

Now the client will always be set, so we don't have to do the headers on each request.